adapter-vercel build fails with TypeError: builder.copy is not a function

**Describe the bug**
adapter-vercel build fails with with TypeError: builder.copy is not a function

"@sveltejs/adapter-vercel": "^1.0.0-next.8",
"@sveltejs/kit": "^1.0.0-next.56",
"svelte": "^3.35.0",

**Logs**
```
> Using @sveltejs/adapter-vercel
> builder.copy is not a function
TypeError: builder.copy is not a function
    at adapt (/Users/harold/code/translate-image/node_modules/@sveltejs/adapter-vercel/index.js:24:12)
    at adapt (file:///Users/harold/code/translate-image/node_modules/@sveltejs/kit/dist/chunks/index5.js:757:8)
    at file:///Users/harold/code/translate-image/node_modules/@sveltejs/kit/dist/cli.js:512:11
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! translate-image@0.0.1 build: `svelte-kit build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the translate-image@0.0.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

**To Reproduce**
1. Brand new SvelteKit repo
2. Replace node adapter with vercel adapter
3. Update dependencies
   "@sveltejs/adapter-vercel": "^1.0.0-next.8",
   "@sveltejs/kit": "^1.0.0-next.56",
   "svelte": "^3.35.0",
4. npm run build

**Expected behavior**
Successfull build

**Stacktraces**
If you have a stack trace to include, we recommend putting inside a `<details>` block for the sake of the thread's readability:

<details>
  <summary>Stack trace</summary>

TypeError: builder.copy is not a function
    at adapt (/Users/harold/code/translate-image/node_modules/@sveltejs/adapter-vercel/index.js:24:12)
    at adapt (file:///Users/harold/code/translate-image/node_modules/@sveltejs/kit/dist/chunks/index5.js:757:8)
    at file:///Users/harold/code/translate-image/node_modules/@sveltejs/kit/dist/cli.js:512:11

</details>

**Information about your SvelteKit Installation:**

- The output of `npx envinfo --system --npmPackages svelte,@sveltejs/kit --binaries --browsers`
```
$ npx envinfo --system --npmPackages svelte,@sveltejs/kit --binaries --browsers
npx: installed 1 in 1.361s

  System:
    OS: macOS 11.2.3
    CPU: (8) x64 Intel(R) Core(TM) i5-1038NG7 CPU @ 2.00GHz
    Memory: 196.52 MB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 12.21.0 - ~/.nvm/versions/node/v12.21.0/bin/node
    Yarn: 1.22.10 - /usr/local/bin/yarn
    npm: 6.14.11 - ~/.nvm/versions/node/v12.21.0/bin/npm
  Browsers:
    Chrome: 89.0.4389.90
    Firefox: 86.0.1
    Safari: 14.0.3
  npmPackages:
    @sveltejs/kit: ^1.0.0-next.56 => 1.0.0-next.56
    svelte: ^3.35.0 => 3.35.0
```
- Your browser
N/A

- Your adapter (e.g. Node, static, Vercel, Begin, etc...)
Vercel

**Severity**
Blocking Entirely

Note: the more honest and specific you are here the more we will take you seriously.

--- a/.changeset/famous-turtles-appear.md
+++ b/.changeset/famous-turtles-appear.md
@@ -1,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+Add copy function to Builder.js

--- a/packages/kit/src/core/adapt/Builder.js
+++ b/packages/kit/src/core/adapt/Builder.js
@@ -37,6 +37,15 @@ export default class Builder {
 		rimraf(path);
 	}
 
+	/**
+	 * @param {string} from
+	 * @param {string} to
+	 * @param {(basename: string) => boolean} filter
+	 */
+	copy(from, to, filter = () => true) {
+		copy(from, to, filter);
+	}
+
 	/** @param {string} dir */
 	mkdirp(dir) {
 		mkdirp(dir);

